src/Concordium/Client/Runner.hs

```diff
@@ -2227,8 +2232,16 @@ processBakerCmd action baseCfgDir verbose backend =
   case action of
     BakerGenerateKeys outputFile -> do
       keys <- generateBakerKeys
-      let out = AE.encodePretty keys
-
+      -- let out = AE.encodePretty keys
+      pwd <- askPassword "Enter password for encryption of baker keys (leave blank for no encryption): "
+      out <- case Password.getPassword pwd of
+        "" -> return $ AE.encodePretty keys
```

Comment:
Perhaps you could log here "empty password, not encrypting baker keys" or similar.